### PR TITLE
Remove job distribution protocol

### PR DESCRIPTION
**Pull request by Shourya742**
Closes: #1065

### CODE REVIEW

File: scripts/build_header.sh [hunk 1]
```diff
@@ -1,9 +1,8 @@
 #! /bin/sh
-
 cargo install --version 0.20.0 cbindgen
 
-rm -f ./scripts/sv2.h
-touch ./scripts/sv2.h
+rm -f ./sv2.h
+touch ./sv2.h
```
Comment (GitGab19) [inline scripts/build_header.sh:5]: Why is this change needed?

Comment (Shourya742) [reply scripts/build_header.sh:5]: We're calling this script from `sv2-header-check.sh`, which changes the PWD to the scripts directory before invoking `build_header.sh`. The buildr_header shell script point to the scripts folder, but since it was already inside the scripts directory, the removal and creation commands couldn't locate the correct path. As a result, it couldn't create the `sv2.h` file at the desired location and instead appended the `cbindgen` output to the older version of `sv2.h`. This caused the hash test to fail.

**Review by GitGab19 — APPROVED** [review]

**Comment by plebhash** [comment]
rebasing so we can merge
